### ISSUE TITLE
Break on last return line twice.

### ISSUE THREAD

**Issue by panzhongxian** [mostafa/goja#14]
@mostafa 

Take the following code as a test case. I leave a blank line here:

```javascript
  1
  2 function test() {
  3   var a = true;
  4   let b = true;
  5   return a;
  6 }
  7 test();
  8 test();
```

When I set a breakpoint on line 8 and I will break there for two times. I think this may be caused by the goja itself send back the last line return value as the `vm.result`.

The walked line tracing record(I have removed continuous repeated records):

```
[case4] current filename: test.js, line: 1
[case4] current filename: test.js, line: 7
[case4] current filename: test.js, line: 1
[case4] current filename: test.js, line: 5
[case4] current filename: test.js, line: 7
[case3] current filename: test.js, line: 8
[case4] current filename: test.js, line: 1
[case4] current filename: test.js, line: 5
[case3] current filename: test.js, line: 8
```

**Comment by mostafa** [comment]
The `pc` is reset inside a function.

**Comment by panzhongxian** [comment]
Closed by #18.
